src/components/layoutFullWidth.jsx

```diff
@@ -9,7 +9,7 @@ export default function Layout({ children, darkBg }) {
     <>
       {/* <Seo></Seo> */}
       <Header />
-      <main className={darkBg && 'darkBg'}>
+      <main className={(darkBg ? 'darkBg' : '')}>
```

Comment:
React was complaining that the className was false

Reply:
Ohh yeah! Expression evaluates to **false**! My bad 😇